xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -112,15 +128,16 @@ struct UserProfileDetailModel: ModelProtocol {
                 environment: environment
             )
            
-        case .populate(let user, let statistics):
+        case .populate(let user, let spherePath, let statistics):
             var model = state
             model.user = user
             model.statistics = statistics
             model.recentEntries = (0...10).map { _ in EntryStub.dummyData(petname: user.petname) }
             model.topEntries = (0...10).map { _ in EntryStub.dummyData(petname: user.petname) }
             model.following = (1...10).map { _ in StoryUser.dummyData() }
```

Comment:
When we use real data we will need to fetch following status for everyone in the list. May be worth a global in-memory cache of Did -> following status, which gets invalidated / updated if you follow and unfollow.